nncf/onnx/graph/nncf_graph_builder.py

```diff
@@ -21,11 +21,167 @@
 from nncf.common.graph.layer_attributes import Dtype
 from nncf.common.graph.operator_metatypes import InputNoopMetatype
 from nncf.common.graph.operator_metatypes import OutputNoopMetatype
-from nncf.onnx.graph.metatypes.onnx_metatypes import ONNX_OPERATION_METATYPES
-from nncf.onnx.graph.metatypes.onnx_metatypes import WEIGHT_LAYER_METATYPES
+from nncf.onnx.graph.metatypes.onnx_metatypes import OPERATIONS_WITH_BIAS_METATYPES
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXConstantMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXDequantizeLinearMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXGemmMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXIdentityMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXQuantizeLinearMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXReshapeMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXTransposeMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import get_bias_tensor_port_id
+from nncf.onnx.graph.metatypes.onnx_metatypes import get_constant_weight_port_ids
+from nncf.onnx.graph.metatypes.onnx_metatypes import get_metatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import get_possible_weight_port_ids
 from nncf.onnx.graph.onnx_graph import ONNXGraph
 
 
+class ONNXLayerAttributes(BaseLayerAttributes):
+    """
+    This class stores attributes of modules/layers for the algorithms.
+    """
+
+    def __init__(
+        self,
+        weight_attrs: Optional[Dict[int, Dict]] = None,
+        bias_attrs: Optional[Dict[str, Any]] = None,
+        node_attrs: Optional[Dict[str, Any]] = None,
+    ):
+        self.weight_attrs = weight_attrs if weight_attrs is not None else {}
+        self.bias_attrs = bias_attrs if bias_attrs is not None else {}
+        self.node_attrs = node_attrs if node_attrs is not None else {}
+
```

Comment:
As far as I remember it was fixed in OV backend
```suggestion
        self.weight_attrs = weight_attrs
        self.bias_attrs = bias_attrs
        self.node_attrs = node_attrs

```